x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/source_selection/source_selection.tsx

```diff
@@ -27,7 +27,13 @@ const fixedPageSize: number = 20;
 
 export const SourceSelection: FC = () => {
   const {
-    services: { http, uiSettings, savedObjectsManagement },
+    services: {
+      http,
+      uiSettings,
+      savedObjectsManagement,
+      data: { dataViews: dataViewsContract },
```

Comment:
actually, now I'm not sure.
they have this renaming type
`export type DataViewsContract = DataViewsServicePublic;`
But they export both `DataViewsContract` and `DataViewsServicePublic`

Reply:
I actually agree that it would be more consistent to rename it to `dataViewsService` to be consistent with other plugins. Looks like DataViewsContract used to be different `PublicMethodsOf<DataViewsServicePublic>`, but has been changed to be the same since [March 2022 (see commit)](https://github.com/elastic/kibana/commit/223068580f84a5472fae08fde9c99fd0260aed37). I think it's safe for us to rename accordingly.